group_vars/all.yml

```diff
@@ -5,3 +5,8 @@ local_remote_user: user
 host_remote_user: user
 stockpile_output_path: /tmp/stockpile.json
 container_cli: docker
+# The temporary directory is used to store json
+# files generated by backpack containers if applicable
+tmp_dir: /tmp/container
```

Comment:
could we possibly rename to backpack_output_path, as this would help folks understand that its not temporary dir.

Reply:
renamed to backpack_output_dir